api/report_server.thrift

```diff
@@ -94,7 +101,9 @@ struct ReportFilter_v2 {
   4: list<string>                 reportHash,
   5: list<shared.Severity>        severity,
   6: list<shared.ReviewStatus>    reviewStatus,
-  7: list<shared.DetectionStatus> detectionStatus
+  7: list<shared.DetectionStatus> detectionStatus,
+  8: optional string              detectedDate,
```

Comment:
Isn't it better to store something like an ID to the run in the history where it was first detected/fixed?

Reply:
Since CodeChecker 6, a run ID is a very, very long running stable variable. A run is only introduced once and then for the entire lifespan of the project, it is not expected to be deleted, only updated (stored into) later on. Storing does not change the run ID.

Reply:
Though I would like it better if dates would be sent as at least integers (in epoch time), not strings, and let the client do the formatting of these values (perhaps different clients with region-specific formats?)

Reply:
I did not mean the run ID, but an ID to an entry in the run history table.